cfgov/v1/util/migrations.py

```diff
@@ -0,0 +1,59 @@
+from django.core.exceptions import ObjectDoesNotExist
+from django.db import transaction
+from treebeard.mp_tree import MP_Node
+
+
+def get_page(apps, slug):
+    base_page_cls = apps.get_model('wagtailcore', 'Page')
+    return base_page_cls.objects.get(slug=slug)
+
+
+def get_free_path(apps, parent_page):
+    offset = 1
+    base_page_cls = apps.get_model('wagtailcore', 'Page')
+
+    while True:
```

Comment:
For spaces, I was looking at the whole file, not the lines you added. The rest is just preference.